code/statistics/commit_key.py

```diff
@@ -0,0 +1,40 @@
+"""
+    Class to contain author info
+"""
+
+
+class CommitKey:
+    def __init__(self, author: str, email: str):
+        """
```

Comment:
Let's use reStructuredText  as docstring style - you can find it in PyCharm when search `docstring format`